{{cookiecutter.project_slug}}/Makefile

```diff
@@ -127,9 +127,9 @@ setup: ### Setup remote environment
 	$(NEURO) mkdir --parents $(PROJECT_PATH_STORAGE) \
 		$(PROJECT_PATH_STORAGE)/$(CODE_DIR) \
 		$(DATA_DIR_STORAGE) \
-		$(PROJECT_PATH_STORAGE)/$(CONFIG_DIR) \
+		$(PROJECT_PATH_STORAGE)/$(CONFIG_DIR)/ \
```

Comment:
Can you please clarify, why most of the paths on the storage and in the container have now '/' at the end, but at the same time some of them don't?

Reply:
Oh, I see now, that this PR is built on top of #380. So the change here seems to be fine.